src/test/cli/rbd/help.t

```diff
@@ -2712,3 +2714,7 @@
     --image arg          image name
   
 
+
```

Comment:
This is what cram generated. I just merged the diff.

Reply:
Did you do it manually or using cram itself?  I'm curious because I have never seen cram do this.

Here is what I get on a checkout of this commit with changes to `src/test/cli/rbd/help.t` discarded and telling cram to interactively merge changed output:
```
$ cram3 -i src/test/cli/rbd/help.t
!
--- src/test/cli/rbd/help.t
+++ src/test/cli/rbd/help.t.err
@@ -1832,19 +1832,21 @@
   rbd help mirror pool enable
   usage: rbd mirror pool enable [--pool <pool>] [--namespace <namespace>] 
                                 [--site-name <site-name>] 
+                                [--remote-namespace <remote-namespace>] 
                                 <pool-spec> <mode> 
   
   Enable RBD mirroring in a pool or namespace.
   
   Positional arguments
-    <pool-spec>          pool specification
-                         (example: <pool-name>[/<namespace>]
-    <mode>               mirror mode [image or pool]
-  
-  Optional arguments
-    -p [ --pool ] arg    pool name
-    --namespace arg      namespace name
-    --site-name arg      local site name
+    <pool-spec>            pool specification
+                           (example: <pool-name>[/<namespace>]
+    <mode>                 mirror mode [image or pool]
+  
+  Optional arguments
+    -p [ --pool ] arg      pool name
+    --namespace arg        namespace name
+    --site-name arg        local site name
+    --remote-namespace arg remote namespace name
   
   rbd help mirror pool info
   usage: rbd mirror pool info [--pool <pool>] [--namespace <namespace>] 
Accept this change? [yN] y
patching file src/test/cli/rbd/help.t

# Ran 1 tests, 0 skipped, 1 failed.
```
`git diff` confirms:
```
$ git diff src/test/cli/rbd/help.t
diff --git a/src/test/cli/rbd/help.t b/src/test/cli/rbd/help.t
index ff25b5973a4..5b7b4774adf 100644
--- a/src/test/cli/rbd/help.t
+++ b/src/test/cli/rbd/help.t
@@ -1832,19 +1832,21 @@
   rbd help mirror pool enable
   usage: rbd mirror pool enable [--pool <pool>] [--namespace <namespace>]
                                 [--site-name <site-name>]
+                                [--remote-namespace <remote-namespace>]
                                 <pool-spec> <mode>

   Enable RBD mirroring in a pool or namespace.

   Positional arguments
-    <pool-spec>          pool specification
-                         (example: <pool-name>[/<namespace>]
-    <mode>               mirror mode [image or pool]
+    <pool-spec>            pool specification
+                           (example: <pool-name>[/<namespace>]
+    <mode>                 mirror mode [image or pool]

   Optional arguments
-    -p [ --pool ] arg    pool name
-    --namespace arg      namespace name
-    --site-name arg      local site name
+    -p [ --pool ] arg      pool name
+    --namespace arg        namespace name
+    --site-name arg        local site name
+    --remote-namespace arg remote namespace name

   rbd help mirror pool info
   usage: rbd mirror pool info [--pool <pool>] [--namespace <namespace>]
```